cluster-operator/src/main/java/io/strimzi/operator/cluster/model/KafkaBridgeConfigurationBuilder.java

```diff
@@ -346,6 +352,53 @@ public KafkaBridgeConfigurationBuilder withHttp(KafkaBridgeHttpConfig http, Kafk
         return this;
     }
 
+    /**
+     * Configures the Strimzi Metrics Reporter. It is set only if user enables Strimzi Metrics Reporter.
+     *
+     * @param model     Strimzi Metrics Reporter configuration
+     *
+     * @return Returns the builder instance
+     */
+    public KafkaBridgeConfigurationBuilder withStrimziMetricsReporter(StrimziMetricsReporterModel model)   {
+        if (model != null) {
+            printSectionHeader("Strimzi Metrics Reporter configuration");
+            writer.println("bridge.metrics=" + TYPE_STRIMZI_METRICS_REPORTER);
+            // the kafka. prefix is required by the Bridge to pass Kafka client configurations
+            writer.println("kafka.metric.reporters=" + StrimziMetricsReporterConfig.KAFKA_CLASS);
+            writer.println("kafka." + StrimziMetricsReporterConfig.LISTENER_ENABLE + "=false");
+            writer.println("kafka." + StrimziMetricsReporterConfig.ALLOW_LIST + "=" + model.getAllowList());
+            writer.println();
+        }
+        return this;
+    }
+
+    /**
+     * Configures the JMX Prometheus Metrics Exporter.
+     *
+     * @param model JMX Prometheus Metrics Exporter configuration
+     * @param isLegacyMetricsConfigEnabled Flag which indicates whether the metrics are enabled or not.
```

Comment:
we should improve "indicates whether the metrics are enabled or not." to be clear they are enabled or not but with the legacy way. Because you can enable them by passing a model != null but the flag as false because you are using the new metrics way.